src/pages/DiaryHome/DiaryHome.tsx

```diff
@@ -0,0 +1,177 @@
+import {ChangeEvent,ChangeEventHandler, FC, FormEvent, useEffect, useState} from "react";
+import {Button, FormControl, Grid, TextareaAutosize, TextField, Typography} from '@mui/material';
+import './DiaryHome.scss'
+import {deepPurple} from "@mui/material/colors";
+import {Link} from "react-router-dom";
+import DiaryCard from "../../components/DiaryCard/DiaryCard";
+import {RootState, store} from "../../store";
+import {useDispatch, useSelector} from "react-redux";
+import {set} from "../SignInForm/userSlice";
+import {add} from "./cardsSlice";
+import {db} from '../../utils/firebaseConfig';
+import {onSnapshot, query,collection, Firestore, getDocs, doc, setDoc, addDoc, DocumentData} from "firebase/firestore";
+import {addId} from "./cardIdsSlice";
+
+type DiaryEntry = {
+    title: string,
+    subtitle: string,
+    description: string,
+    color: string
+}
+
+const DieryHome: FC = () => {
+
+    const [inputCardTitle, setInputCardTitle] = useState<string>('');
+    const [inputCardDescription, setInputCardDescription] = useState<string>('');
+    const [loading, setLoading] = useState<boolean>(true);
+
+    let cards = useSelector((state: RootState) => state.cards.value);
+
+    let cardIds = useSelector((state: RootState) => state.cardIds.value);
+
+    // console.log("At begining")
+    // console.log(cardIds.length)
+
+    async function getCards(db: Firestore) {
+        // console.log("At get card begining")
+        // console.log(cardIds.length)
+        const q = query(collection(db, "Cards"));
+        const unsubscribe = onSnapshot(q, (querySnapshot) => {
+            querySnapshot.forEach((doc) => {
+                if(!cardIds.includes(doc.id)){
```

Comment:
remove unnecessary console.logs when commit the code

Reply:
resolved in the Handling Issues(https://github.com/fcodelabs/training-web/pull/251) PR